_rules/video-only-transcript-ee13b5.md

```diff
@@ -36,19 +43,19 @@ This rule applies to any [non-streaming](#non-streaming-media-element) `video` e
 
 ## Expectation
 
-The visual information of each test target is available through a text transcript that is [visible][], [included in the accessibility tree][], and is either on the page or linked.
+The visual information of each test target is available through a text transcript that is [visible][], and is either on the page or in a clearly labeled location.
 
 ## Assumptions
 
-A mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster).
+A mechanism is available to start the video and that the video element is not simply used to display the [poster](https://www.w3.org/TR/html5/semantics-embedded-content.html#element-attrdef-video-poster). This rule also assumes that if a transcript is present it is [included in the accessibility tree][].
```

Comment:
Putting it as an assumption for something that is so easily testable sounds weird 🤔

Reply:
This was what we decided was the best way to fix the concern. Basically, we are not testing the fact it is in the accessibility tree for 1.2.1 but assume it is there.